plugins/modules/keycloak_client.py

```diff
@@ -775,8 +775,9 @@ def sanitize_cr(clientrep):
     if 'secret' in result:
         result['secret'] = 'no_log'
     if 'attributes' in result:
-        if 'saml.signing.private.key' in result['attributes']:
-            result['attributes']['saml.signing.private.key'] = 'no_log'
+        attributes = result['attributes']
+        if 'saml.signing.private.key' in attributes and isinstance(attributes, dict):
```

Comment:
I would switch the order, so it never tries to test membership if it isn't a dict:
```suggestion
        if isinstance(attributes, dict) and 'saml.signing.private.key' in attributes:
```

Reply:
Makes sense, I adapted your suggestion.